test/Microsoft.AspNetCore.ResponseCaching.Tests/TestUtils.cs

```diff
@@ -15,6 +15,7 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Internal;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
```

Comment:
Is this the only change needed because it was picking up transitive dependencies from somewhere?